README.md

```diff
@@ -30,6 +30,18 @@ in another terminal if you are doing front end work:
 
     gulp watch
 
+## Running common tasks
+
+When running migrations, make sure you set up a SECRET_KEY in a .env file for yourself locally
```

Comment:
Is the `.env` file loaded automatically by the Django app?

Reply:
The .env setting is for having a variables available in docker during the pre-processing step, so having the secret key set there helps with running commands via docker. Variables in the env file are not available to the app after the build.

Reply:
Like we talked about — `SECRET_KEY` is supposed to be defined automatically in the local environment by a function in `local_settings.py`, but that is not working for me. The README suggests exporting a secret key manually. And this introduces a third path to setting a secret key. We should consolidate down to a single path for setting a secret key and make sure that works properly in the local environment for all devs.